core/src/main/resources/log4j2.xml

```diff
@@ -9,7 +9,7 @@
     <Logger name="kafka.consumer" level="INFO" additivity="false">
       <AppenderRef ref="primary" />
     </Logger>
-    <Logger name="com.spotify.ffwd" level="WARN" additivity="false">
+    <Logger name="com.spotify.ffwd" level="INFO" additivity="false">
```

Comment:
does this produce a lot of logs? iirc we changed this to reduce the logging in production. perhaps you can add a logger just for the class you are interested in instead?

Reply:
reverting it back, just missed it